app/resources/api/v2/tag_set_resource.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # @todo This documentation does not yet include a detailed description of what this resource represents.
+    # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
+    # @todo This documentation does not yet include any example usage of the API via cURL or similar.
```

Comment:
Would you have a stab at filling these out? It seems like it will be easiest to do right now when we have the model and reasons for creating it fresh in our minds.

Reply:
Yep sure, im not actually sure if I need this resource as I haven't exposed the TagSet in the API, I will check. Should I be exposing TagSets in the API? Is that something I should do in this PR or should that be included if and when required (e.g. needed in a Limber story).

Reply:
API will be covered by another story so I am going to remove resource references from this one.